### DISCUSSION TITLE
Milestones in the link section

### DISCUSSION THREAD

**Discussion by CBID2** [EddieHubCommunity/LinkFree#7692] [Ideas]
Hi everyone! :) What do y'all think of creating a feature where milestones could be displayed in the links section with a checkbox with the same fields?  @eddiejaoude came up with this idea while I was reviewing this [PR](https://github.com/EddieHubCommunity/LinkFree/pull/7684#discussion_r1234681988).

**Comment by eddiejaoude** [comment]
Some of my thoughts...
- note that only milestones with urls can have this feature (toggle)
- maybe the group name for these could be `Milestones`

**Reply by CBID2** [reply]
I agree with the first bullet point. As far as giving the group name `Milestones`, I partially agree due to the consistency, but I think `Achievements` would be a good group name too. What do you think?
